packages/sanity/src/core/hooks/useGlobalCopyPasteElementHandler.ts

```diff
@@ -1,8 +1,9 @@
 import {type Path} from '@sanity/types'
 import {isHotkey} from 'is-hotkey-esm'
 import {useCallback, useEffect, useRef} from 'react'
-import {type FormDocumentValue, useCopyPasteAction} from 'sanity'
+import {type FormDocumentValue} from 'sanity'
```

Comment:
I'm noticing a few imports from `sanity` here. Do we prefer to import from local sources, e.g., `/core/form/types`?

Reply:
ooo great find